src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs

```diff
@@ -82,6 +74,23 @@ internal static void Register(CommandLineApplication app)
             });
         }
 
+        /// <summary>
+        /// Internal for testing.
+        /// </summary>
+        public static ProjectContext ResolveProjectContext(string projectFilePath)
```

Comment:
Would rather not have an `InternalsVisibleTo` and no one else consumes this :smile: